e2e/playwright/info-drawer/src/tests/general.spec.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test, TrashcanApi, NodesApi, FileActionsApi } from '@alfresco/playwright-shared';
+
+test.describe('Info Drawer - General', () => {
+  let nodesApi: NodesApi;
+  let trashcanApi: TrashcanApi;
+  let fileActionsApi: FileActionsApi;
+
+  const username = `user1-${Utils.random()}`;
+
+  test.beforeAll(async () => {
+    try {
+      const apiClientFactory = new ApiClientFactory();
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username });
+      nodesApi = await NodesApi.initialize(username, username);
+      trashcanApi = await TrashcanApi.initialize(username, username);
+      fileActionsApi = await FileActionsApi.initialize(username, username);
+    } catch (error) {
+      console.error(`beforeAll failed: ${error}`);
+    }
+  });
+
+  test.afterAll(async () => {
+    try {
+      await nodesApi.deleteCurrentUserNodes();
+      await trashcanApi.emptyTrashcan();
+    } catch (error) {
+      console.error(`afterAll failed: ${error}`);
+    }
+  });
+
+  test.beforeEach(async ({ loginPage }) => {
+    try {
+      await loginPage.loginUser({ username, password: username }, { withNavigation: true, waitForLoading: true });
+    } catch (error) {
+      console.error(`beforeEach failed: ${error}`);
+    }
+  });
+
+  test.afterEach(async ({ personalFiles }) => {
```

Comment:
afterEach block does nothing here, as it does not continue working in the same web context as the test. Also, afterEach block should only be used for test cleanup (files deleting etc), not for test actions. Test actions should be only performed within the test block

This block can be deleted or changed to test cleanup in my opinion.